addons/web_editor/static/src/js/editor/odoo-editor/src/utils/sanitize.js

```diff
@@ -144,7 +144,7 @@ class Sanitize {
                     )
                 ) &&
                 !isBlock(node.parentElement) &&
-                anchor !== node
+                (!anchor || anchor !== node)
```

Comment:
If `anchor` and `node` are equal while `anchor` is falsy then the line `node.textContent.includes('\u200B')` should crash, right ? So `node` cannot be falsy at that point. So when `anchor` is falsy but `node` is truthy, the current condition would return `true` regardless of `anchor` being falsy, which is exactly what it is doing after your change. So why the change in the first place ? Did I miss something ?